src/plugins/intel_gpu/tests/test_cases/binary_convolution_gpu_test.cpp

```diff
@@ -420,6 +428,9 @@ TEST(binary_convolution, basic_convolution_1x1_single_packed_channel) {
 
 TEST(binary_convolution, basic_convolution_1x1_single_packed_channel_fp16) {
     auto& engine = get_test_engine();
+    // DG2 has an issue for binary convolution: https://github.com/openvinotoolkit/openvino/pull/12486
```

Comment:
Let's use soft expression: `not validated for binary convolution`. Please don't mention product name directly here.

Reply:
Done.